src/components/MDX/Card.styled.ts

```diff
@@ -0,0 +1,24 @@
+import {styled} from "@styles/stitches.ts";
+
+const CardStyled = styled("div", {
+  paddingBottom: "$gr4",
+  zIndex: "1",
+
+  "@xxs": {
+    paddingBottom: "$gr2",
+  },
+
+  "@xs": {
+    paddingBottom: "$gr2",
+  },
+
+  "@sm": {
+    paddingBottom: "$gr3",
+  },
+
+  "@md": {
+    paddingBottom: "$gr3",
+  },
+});
```

Comment:
Two things here, we don't need the in between steps, just when things change. And also, we'll want to step down (md --> xs) rather than up to properly capture the responsive `max-width` breakpoints.

```suggestion
  "@md": {
    paddingBottom: "$gr3",
  },

  "@xs": {
    paddingBottom: "$gr2",
  },
```